lib/LockManager.php

```diff
@@ -125,7 +125,7 @@ public function isLocked(int $id, string $token): bool {
 		$userRoot = $this->rootFolder->getUserFolder($user->getUID());
 		$nodes = $userRoot->getById($id);
 		foreach ($nodes as $node) {
-			while ($node->getPath() !== '/') {
+			while ($node->getPath() !== $userRoot->getPath()) {
```

Comment:
is this leading/trailing slashes-safe ?

Reply:
It's using the node api and the path are cleaned by it